frontend/ui/data/keyboardlayouts/generic_ime.lua

```diff
@@ -56,8 +57,11 @@ local IME = {
     show_candi_callback = function() end,
     switch_char = "下一字", -- default
     separator = "分隔",  -- default
```

Comment:
Should these stay in a generic IME ?
Does the vietnamese keyboard show them ?

Reply:
> that would mean rebase and force push yet again! 

Nothing wrong with that, we all do that and many times per PR when we have multiple topic commits.
But as you wish.

Reply:
If that's the case, then let me force push again. I just thought the many not-so-meaningful push records might be too distracting.